src/sonic-yang-models/tests/yang_model_tests/yangModelTesting.py

```diff
@@ -221,7 +246,7 @@ def runExceptionTest(self, test):
             self.logStartTest(desc)
             jInput = self.readJsonInput(test)
             # load the data, expect a exception with must condition failure
-            s = self.loadConfigData(jInput)
+            s = self.loadConfigData(jInput, self.ExceptionTests[test].get('verify'))
             eStr = self.ExceptionTests[test]['eStr']
             log.debug(eStr)
             if (sum(1 for str in eStr if str not in s) == 0):
```

Comment:
loadConfigData() returns 'verified' when success,  can you check  'verified' as PASS?

Reply:
@li-pingmao :  Yes, I am checking that here. All expected strings in eStr must be in s. eStr == [Verified] in this case.